### PR TITLE
Completes attributions

### PR DESCRIPTION
**Pull request by mkbhanda**
1) Updates Makefile to include a test target
2) Includes go mod tidy and attributions test scripts (copied over from app-service-configurable)
3) Updates some go files per gofmt prescription
4) Adds Attributes.txt file that includes attributions per bin/test-attribution-txt.sh

This should address issue: #287

### CODE REVIEW

**Comment by difince** [comment]
@mkbhanda You missed to sign your commits https://github.com/edgexfoundry/edgex-cli/pull/291/checks?check_run_id=1146548916

**Comment by difince** [comment]
@mkbhanda could you rebase, so the PR contains just your changes

File: cmd/device/device.go [hunk 1]
```diff
@@ -1,39 +0,0 @@
...
-	"github.com/edgexfoundry/edgex-cli/cmd/device/add"
-	"github.com/edgexfoundry/edgex-cli/cmd/device/list"
-	"github.com/edgexfoundry/edgex-cli/cmd/device/rm"
-	"github.com/edgexfoundry/edgex-cli/cmd/device/update"
-	
-	"github.com/spf13/cobra"
-)
-
-// NewCommand returns the device command of type cobra.Command
-func NewCommand() *cobra.Command {
```
Comment (difince) [inline cmd/device/device.go:28]: This file `cmd/device/device.go` should be present that is why 
make test fails with 

> go: finding module for package github.com/edgexfoundry/edgex-cli/cmd/device
> cmd/root.go:25:2: no matching versions for query "latest"
> Makefile:29: recipe for target 'test' failed
> make: *** [test] Error 1

Comment (difince) [reply cmd/device/device.go:28]: Rebase could help with this issue and will make the PR contains just the files you have really changed

**Review by difince — CHANGES_REQUESTED** [review]

**Comment by mkbhanda** [comment]
Thank you @difince for catching the issues. Addressed them.

**Review by difince — APPROVED** [review]

File: cmd/device/device.go (hunk 1 above)
Comment (difince) [reply cmd/device/device.go]: make test still fails with me ..but I doubt that the problem is in my local environment. I will mege the PR and later investigate why  make test do not succeed with me.

**Pull request merged**